src/dataflow/src/render/sources.rs

```diff
@@ -193,13 +191,6 @@ where
                     _ => None,
                 };
 
-                let fast_forwarded = match &connector {
```

Comment:
detecting fast forwarding was there to prevent users from processing debezium data from a topic that was consumed starting from an offset. This is removed in this PR because it was Kafka specific and also there might be legitimate reasons for a user to start their kafka source from an offset. See also this comment https://github.com/MaterializeInc/materialize/pull/9937#discussion_r782015254

Reply:
Is that link broken?